website/src/pages/subscriptions/create.astro

```diff
@@ -18,150 +18,182 @@ const variantFilter = {
+                                    </p>
+                                    <div class='w-full'>
+                                        <gs-location-filter
+                                            id='locationFilter'
+                                            fields={JSON.stringify(locationFilterFields)}
+                                            initialValue='Europe / Switzerland'
+                                            placeholderText='Sampling location'
+                                            width='100%'></gs-location-filter>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div>
+                                <div>
+                                    <h3 class='mb-2'>Variant</h3>
+                                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                                        The variant you want to receive notifications for.
+                                    </p>
+                                    <div class='flex w-full flex-col gap-2'>
+                                        <gs-text-input
+                                            id={`textFilter`}
+                                            lapisField={'host'}
+                                            placeholderText={'Some text input'}
+                                            width='100%'></gs-text-input>
+                                        <gs-mutation-filter id='mutationFilter' width='100%'></gs-mutation-filter>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class='min-w-72 flex-1'>
+                            <div>
+                                <h2 class='component-title'>Prevalence over time</h2>
+                                <div class='h-[400px]'>
+                                    <gs-prevalence-over-time
+                                        numeratorFilter={JSON.stringify({
+                                            displayName: '',
+                                            lapisFilter: variantFilter,
+                                        })}
+                                        denominatorFilter={JSON.stringify(baselineFilter)}
+                                        granularity='month'
+                                        smoothingWindow='0'
+                                        width='100%'
+                                        height='100%'></gs-prevalence-over-time>
+                                </div>
+                            </div>
                         </div>
                     </div>
-                </div>
+                </gs-app>
+            </div>
+
+            <div class='flex w-full'>
+                <button class='btn w-1/2'>Cancel</button>
```

Comment:
I wonder - what would that button actually do? Should we simply remove it?

Reply:
I would link back to the `/subscriptions` page. But we could also add a link to that somewhere and remove this button here?